server/base/src/main/java/org/apache/accumulo/server/fs/VolumeChooserEnvironment.java

```diff
@@ -35,17 +38,37 @@
   private final ServerContext context;
   private final ChooserScope scope;
   private final Table.ID tableId;
+  private final TabletId tabletId;
 
   public VolumeChooserEnvironment(ChooserScope scope, ServerContext context) {
     this.context = context;
     this.scope = Objects.requireNonNull(scope);
     this.tableId = null;
+    tabletId = null;
+  }
+
+  @VisibleForTesting
+  public VolumeChooserEnvironment(Table.ID tableId) {
```

Comment:
I do not think we need this method. I believe it was removed from the master branch. Might be useful to rebase yours.

Reply:
Fixed in next commit